README.md

```diff
@@ -21,11 +21,40 @@
 
 # AutoGen
 
-[📚 Cite paper](#related-papers).
-<!-- <p align="center">
-    <img src="https://github.com/microsoft/autogen/blob/main/website/static/img/flaml.svg"  width=200>
-    <br>
-</p> -->
+
+
+
+AutoGen is an open-source programming framework for building AI agents and facilitating cooperation among multiple agents to solve tasks. AutoGen aims to streamline the development and research of agentic AI, much like PyTorch does for Deep Learning. It offers features such as agents capable of interacting with each other, facilitates the use of various large language models (LLMs) and tool use support, autonomous and human-in-the-loop workflows, and multi-agent conversation patterns.
+
+
+> *Note for contributors and users*</b>: [microsoft/autogen](aka.ms/autogen-gh) is the official repository of AutoGen project and it is under active development and maintenance under MIT license. We welcome contributions from developers and organizations worldwide. Our goal is to foster a collaborative and inclusive community where diverse perspectives and expertise can drive innovation and enhance the project's capabilities. We acknowledge the invaluable contributions from our existing contributors, as listed in [contributors.md](./CONTRIBUTORS.md). Whether you are an individual contributor or represent an organization, we invite you to join us in shaping the future of this project. For further information please also see [Microsoft open-source contributing guidelines](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing).
```

Comment:
The link "aka.ms/autogen-gh" doesn't work, it goes to "https://github.com/microsoft/autogen/blob/update-readme/aka.ms/autogen-gh". Maybe use "https://aka.ms/autogen-gh".

Reply:
Sorry about this and thank you for catching it! I fixed it.